Gaijinizer patch/Map059.txt

```diff
@@ -332,139 +347,146 @@ Well, that smile is scary in another way ......!
 ワシはさっきも言ったように追放された身じゃけん……
 個人で莫大な研究費用とかを稼がないかんのよ……
 > CONTEXT: Map059/events/1/pages/0/223/Dialogue/other3
-Well, yeah, The eagle was exiled as I said before
-... I do not earn huge research expenses
-individually ...#MTLed
+Well, you see... Like I said before I was exiled,
+and I had to find a way to cover all my huge
+research expenses...#Ron
 > END STRING
 
 > BEGIN STRING
 だから、人間のボンボンに高く売って金儲けしようって
 
 ワシが密かに開発してた『搾精人形』なんよ……アレ。
 > CONTEXT: Map059/events/1/pages/0/228/Dialogue/other3
-So I'm trying to make money by selling high for
-human sweets. The "milking doll" secretly
-developed by the eagle...What.#MTLed
+So I told everyone I was trying to sell human
+candy for a high price but secretly I was working
+on that milking doll...#Ron
 > END STRING
 
 > BEGIN STRING
 でも、完成間近って時になぁ……
 
 べらぼうに強い怨念が人形に執り憑いてしまってん……。
 > CONTEXT: Map059/events/1/pages/0/232/Dialogue/other3
-But when it's almost finished ...
-I have a strong condolences over the dolls.#MTLed
+But then when I was almost finished...
+
+It got possessed by some extreme grudge and became
+haunted.#Ron
 > END STRING
 
 > BEGIN STRING
 『怨念』……？
 > CONTEXT: Map059/events/1/pages/0/236/Dialogue/kazuya
-"Plenary" ...?#MTLed
+"Grudge"...?#Ron
 > END STRING
 
 > BEGIN STRING
 幽霊みたいなもんや…
 
 しかもありゃあ相当の業を背負ってるレベルやった…
 > CONTEXT: Map059/events/1/pages/0/238/Dialogue/other3
-It looks like a ghost...Besides, I did a level
-of carrying a considerable amount of work ...#MTLed
+It seems to be a ghost of some kind...
+
+And I'd put so much work into it as well...#Ron
 > END STRING
 
 > BEGIN STRING
 しかもその怨念、人形が『それ目的』なのに気づいてな…
 
 ワシに『男はどこだ』って脅してきたから、とっさに…
 > CONTEXT: Map059/events/1/pages/0/242/Dialogue/other3
-Moreover, I realize that the puppet is "purpose"
-... As I threatened to the eagle "Who is the
-man?"#MTLed
+And I think you've realised the "purpose" of that
+doll's grudge... She menacingly asked "Where is the
+man" and then suddenly...#Ron
```

Comment:
Flow:
>_**By now I think you've realized the reason
behind that doll's grudge... She menacingly asked,
"Where is that man?" and then suddenly...**_